docs/sources/write/_index.md

```diff
@@ -1,29 +1,31 @@
 ---
-title: Write documentation
-menuTitle: Write
-description: Learn about Grafana Labs' technical documentation writing guidelines.
-weight: 400
 aliases:
   - /docs/writers-toolkit/writing-guide/
   - /docs/writers-toolkit/write/
-Keywords:
+date: 2024-02-29
+description: Learn about Grafana Labs' technical documentation writing guidelines.
+keywords:
   - writing
   - guidelines
+menuTitle: Write
+title: Write documentation
+weight: 400
 ---
 
 # Write documentation
 
-This writing guide defines the structured authoring environment we use to create documentation at Grafana Labs.
+This writing guide defines how Grafana Labs writes documentation.
 If you write technical documentation for Grafana Labs, familiarize yourself with these guidelines.
 
-We write our documentation using [U.S. English](https://developers.google.com/style/translation).
+## Before you write
```

Comment:
I thought instead of this page containing one rule (write in U.S. English), it's better for this page to link and summarize the sections where all the rules are and other things to consider before you start.